fastly/validators.go

```diff
@@ -140,3 +141,16 @@ func validateHTTPSURL() schema.SchemaValidateFunc {
 		return
 	}
 }
+
+func validatePEMBlock(pemType string) schema.SchemaValidateFunc {
+	return func(val interface{}, key string) ([]string, []error) {
+		b, _ := pem.Decode([]byte(val.(string))) // TODO: might want to loop here to support multiple PEM resources in one string for intermediates blob?
```

Comment:
Outstanding question here about what the contents of the `intermediates_blob` should be. Currently only validates for one certificate but maybe there could be multiple? I asked Alexander on Friday who said he'd check

Reply:
I've just had a response - the suspicion was correct in that there can be multiple PEM blocks in the `intermediates_blob`. Will need to update to account for this, either as an extra on this PR or a followup depending on timings